### PR TITLE
Retrieve tags

### PR DESCRIPTION
**Pull request by vnegrisolo**
- [x] new service to fetch `async` results from keepcon processing

### CODE REVIEW

File: lib/keepcon/client.rb [hunk 1]
```diff
@@ -4,36 +4,42 @@ class Client
...
     end
 
-    def content_request(data, mode = :sync)
+    def content_request(data, mode)
       fail ArgumentError, 'The data can not be empty' unless data.present?
 
       request(:put, config['urls']['content']['request'][mode.to_s], data, mode)
     end
 
+    def fetch_results
```
Comment (ggcampinho) [inline lib/keepcon/client.rb]: Can you rename to `async_results_request`?

File: spec/integration_spec.rb [hunk 2]
```diff
@@ -95,4 +95,25 @@
...
     end
   end
+
+  describe '.fetch_keepcon_results' do
+    subject { dummy_class.fetch_keepcon_results(context) }
+
+    let(:results) { double status: 200, body: 'keepcon-xml-response' }
+
+    context 'when context do not exists' do
+      let(:context) { :test }
```
Comment (ggcampinho) [inline spec/integration_spec.rb]: Can you rename `:test` to `:non_existent`

**Comment by vnegrisolo** [comment]
@ggcampinho I did the renaming job.. Could you verify again please?

**Comment by brennovich** [comment]
:+

**Comment by brennovich** [comment]
:+1:

**Comment by ggcampinho** [comment]
:+1:

**Pull request merged**